src/EditorFeatures/CSharpTest/GenerateVariable/GenerateVariableTests.cs

```diff
@@ -4078,18 +4078,6 @@ static void Goo(IEnumerable<string?> s)
 index: LocalIndex);
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateVariable)]
-        public async Task TestLocalMissingForVar()
-        {
-            await TestMissingInRegularAndScriptAsync(
-@"class Program
-{
-    void Main()
-    {
-        var x = [|var|];
-    }");
-        }
```

Comment:
Is this test deleted and not moved somewhere below? I think it's better to update its name and the expected result of it, instead of completely deleting.

Reply:
yes. please never delete tests.  keep them and show how their behavior changed so we can determine if that behavior is desirable.  Thanks!

Reply:
Moved it down to other tests of contextual keywords.

Reply:
in this case, it's ok given that it *appears* (though is hard to verify) that th exact scenario is covered.  In general though, please do not move or delete existing tests (or touch the *input* to a test).  Only update title, or new expected output if for the new behavior.

Reply:
> in this case, it's ok

> please do not move or delete existing tests

So should I move it to the place it was, or what?

Reply:
in this case, it's ok (but please try to avoid this from now on in all future PRs).